engine/runner/src/main/scala/org/enso/runner/Main.scala

```diff
@@ -322,6 +327,54 @@ object Main {
     exitSuccess()
   }
 
+  /** Handles the `--docs` CLI option.
+    *
+    * Generates reference website from standard library.
+    *
+    * @param projectPath if specified, the docs is generated for a project
+    *                    at the given path
+    * @param logLevel log level to set for the engine runtime
+    */
+  private def genDocs(
+    projectPath: Option[String],
+    logLevel: LogLevel
+  ): Unit = {
+    val stdLibPath   = "./distribution/std-lib"
+    val tempFileName = "Geo/src/Geo_Json.enso"
+    val testFilepath = stdLibPath + "/" + tempFileName
+    val path         = projectPath.getOrElse(testFilepath)
+    generateFrom(path, logLevel)
+    exitSuccess()
+  }
+
+  def generateFrom(path: String, logLevel: LogLevel): Unit = {
```

Comment:
I think this should say that it generates docs, because this method is a general-purpose module so that is not implicitly implied.